.github/workflows/validate-pull-request.yml

```diff
@@ -0,0 +1,27 @@
+name: MPSDK Pull Request Validator
+
+on:
+  pull_request:
```

Comment:
I wonder if we should also add it to check on a push to main also? Just to make sure nothing broke post merge?

```
push:
    branches:
      - main
```

Reply:
Yeah i'm down to add that as well!